_benchmark/user-guide/concepts.md

```diff
@@ -25,6 +25,93 @@ A workload is a specification of one or more benchmarking scenarios. A workload
 - One or more data streams that are ingested into indexes.
 - A set of queries and operations that are invoked as part of the benchmark.
 
+## Throughput and latency
+
+At the end of each test, OSB produces a table that summarizes the following 
+
+- [Service time](#service-time) 
+- Throughput
+- [Latency](#latency)
+- The error rate for each task or OpenSearch operation that ran 
+
+While the definition for _throughput_ remains consistent with other client-server systems, the definitions for `service time` and `latency` differ from most client-server systems in the context of OSB. The following table compares how service time and latency work with OSB versus the common definition of each term for a client-server system.
+
+| Metric | Common definition | **OSB Definition**	|
+| :--- | :--- |:--- |
+| **Throughput** | The number of operations completed in a given period of time	| The number of operations completed in a given period of time	|
+| **Service time**	|The time the server takes to process a request, from the point it receives the request to the point the response is returned. </br></br> It includes the time spent waiting in server-side queues, but _excludes_ network latency, load-balancer overhead, and deserialization/serialization. | The time it takes for the `opensearch-py` to send a request to and receive a response from the OpenSearch cluster. </br> </br> It includes the time it takes for the server to process a request, and also _includes_ network latency, load-balancer overhead, and deserialization/serialization.  |
+| **Latency** | The total time, including the service time and the time the request waited before responding. | The total the request waited before responding based on the `target-throughput` set and any other delays that occur before the request responds. |
+
+
+### Service time
+
+OSB does not have insight into how long OpenSearch takes to process a request, apart from extracting the `took` time for the request. In OpenSearch, **service time** tracks the durations between the point OpenSearch issues a requests and when OpenSearch receives the response.
+
+OSB makes function calls to `opensearch-py`, a client for OpenSearch, to communicate with an OpenSearch cluster. OSB tracks the time between when the `opensearch-py` client sends the request and receives a response from the OpenSearch cluster and considers this to be the service time. Contrasting with the traditional definition of service time, OSB’s definition of service time includes overhead, such as network latency, load-balancer overhead, or deserialization/serialization. The following image illustrates how OpenSearch measure service time versus the traditional measurement. 
```

Comment:
`measure` -> `measures`

Reply:
Also let's rephrase `The following image illustrates how OpenSearch measure service time versus the traditional measurement.`
to
`The following image highlights the difference between traditional service time and OSB's view of service time`